clickhouse_search/models.py

```diff
@@ -581,6 +581,310 @@ class ClinvarMito(BaseClinvarJoin):
+        )
+
+class HgmdSeqrVariantsGRCh37SnvIndel(BaseHgmd):
+    key = OneToOneField('AnnotationsGRCh37SnvIndel', db_column='key', primary_key=True, on_delete=CASCADE)
+
+    class Meta:
+        db_table = 'GRCh37/SNV_INDEL/reference_data/hgmd/seqr_variants'
+        engine = models.MergeTree(
+            primary_key=('key'),
+            order_by=('key'),
+        )
+
+class HgmdSeqrVariantsSnvIndel(BaseHgmd):
+    key = OneToOneField('AnnotationsSnvIndel', db_column='key', primary_key=True, on_delete=CASCADE)
+
+    class Meta:
+        db_table = 'GRCh38/SNV_INDEL/reference_data/hgmd/seqr_variants'
+        engine = models.MergeTree(
+            primary_key=('key'),
+            order_by=('key'),
+        )
+
+class HgmdGRCh37SnvIndel(BaseHgmd):
+    key = ForeignKey('EntriesGRCh37SnvIndel', db_column='key', related_name='hgmd_join', primary_key=True, on_delete=PROTECT)
+
+    class Meta():
+        db_table = 'GRCh37/SNV_INDEL/reference_data/hgmd'
+        engine = Join('ALL', 'LEFT', 'key', join_use_nulls=1, flatten_nested=0)
+
+class HgmdSnvIndel(BaseHgmd):
+    key = ForeignKey('EntriesSnvIndel', db_column='key', related_name='hgmd_join', primary_key=True, on_delete=PROTECT)
+
+    class Meta():
+        db_table = 'GRCh38/SNV_INDEL/reference_data/hgmd'
+        engine = Join('ALL', 'LEFT', 'key', join_use_nulls=1, flatten_nested=0)
+
+class BaseTopmed(models.ClickhouseModel):
+    ac = models.UInt32Field()
+    af = models.DecimalField(max_digits=9, decimal_places=8)
+    an = models.UInt32Field()
+    het = models.UInt32Field()
+    hom = models.UInt32Field()
+
+    class Meta:
+        abstract = True
+
+class TopmedAllVariantsGRCh37SnvIndel(BaseTopmed):
+    variant_id = models.StringField(db_column='variantId', primary_key=True)
+
+    class Meta:
+        db_table = 'GRCh37/SNV_INDEL/reference_data/topmed/all_variants'
+        engine = models.MergeTree(
+            primary_key=('variant_id'),
+            order_by=('variant_id'),
+        )
+
+class TopmedAllVariantsSnvIndel(BaseTopmed):
+    variant_id = models.StringField(db_column='variantId', primary_key=True)
+
+    class Meta:
+        db_table = 'GRCh38/SNV_INDEL/reference_data/topmed/all_variants'
+        engine = models.MergeTree(
+            primary_key=('variant_id'),
+            order_by=('variant_id'),
+        )
+
+class TopmedSeqrVariantsGRCh37SnvIndel(BaseTopmed):
+    key = OneToOneField('AnnotationsGRCh37SnvIndel', db_column='key', primary_key=True, on_delete=CASCADE)
+
+    class Meta:
+        db_table = 'GRCh37/SNV_INDEL/reference_data/topmed/seqr_variants'
+        engine = models.MergeTree(
+            primary_key=('key'),
+            order_by=('key'),
+        )
+
+class TopmedSeqrVariantsSnvIndel(BaseTopmed):
+    key = OneToOneField('AnnotationsSnvIndel', db_column='key', primary_key=True, on_delete=CASCADE)
+
+    class Meta:
+        db_table = 'GRCh38/SNV_INDEL/reference_data/topmed/seqr_variants'
+        engine = models.MergeTree(
```

Comment:
Wouldn't we want the seqr tables to be rocksdb or join tables? Doing  a dozen joins on regular clickhouse tables feels like it would be prohibitively slow